homepage.html

```diff
@@ -0,0 +1,313 @@
+
+
+
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="UTF-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1.0">
+       <title>Contributing to Open Source Part I: The Easy Way </title>
+       <script src="https://kit.fontawesome.com/e53eea7683.js" crossorigin="anonymous"></script>
+       <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+       <link rel="stylesheet" href="./homepage.css">
+   </head>
+
+   <body>
+     <nav>
+       <section class="menu-logo">
```

Comment:
## Check usage of semantic HTML elements. 
Section tag need a h2-h6 tag to accomplish. Check html [validator](https://validator.w3.org/#validate-by-input)